test/Api.IntegrationTest/SecretsManager/Controllers/AccessPoliciesControllerTests.cs

```diff
@@ -167,12 +168,13 @@ public async Task CreateProjectAccessPolicies_Success(PermissionType permissionT
     }
 
     [Theory]
-    [InlineData(false, false)]
-    [InlineData(true, false)]
-    [InlineData(false, true)]
-    public async Task UpdateAccessPolicy_SmNotEnabled_NotFound(bool useSecrets, bool accessSecrets)
+    [InlineData(false, false, false)]
+    [InlineData(true, false, false)]
+    [InlineData(false, true, false)]
+    [InlineData(false, false, true)]
```

Comment:
```suggestion
    [InlineData(false, false, false)]
    [InlineData(false, false, true)]
    [InlineData(false, true, false)]
    [InlineData(false, true, true)]
    [InlineData(true, false, false)]
    [InlineData(true, false, true)]
    [InlineData(true, true, false)]
```

A couple of more cases to test for each of these to cover everything.